### PR TITLE
HTMLEntitiesDecode: Converting to Text Template

### PR DESCRIPTION
**Pull request by mintsoft**
In reference to: #2767 ; I've not tested it in `duckpan server` yet as I can't run it from my current location.

--------------------------------
Instant Answer Page: https://duck.co/ia/view/htmlentities_decode

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @nishanths

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/HTMLEntitiesDecode.t [hunk 1]
```diff
@@ -55,11 +63,12 @@ ddg_goodie_test(
     'html decode apostrophe' => undef,
 
     # natural querying
-    'What is the decoded html entity for &#960;?' => test_zci(qr/.*/, structured_answer => any_structured_answer()),
+    'What is the decoded html entity for &#960;?' => test_zci(qr/.*/, structured_answer => build_structured_answer("&pi;","HTML Entity Decode: &amp;#960;")),
```
Comment (GuiltyDolphin) [inline t/HTMLEntitiesDecode.t]: We should probably hard-code the title.

File: t/HTMLEntitiesDecode.t [hunk 2]
```diff
@@ -8,44 +8,52 @@ use DDG::Test::Goodie;
...
+	    }
 	};
 }
 
 ddg_goodie_test(
     [qw(DDG::Goodie::HTMLEntitiesDecode)],
 
     # Simple decimal test
-    'html decode &#33;' => test_zci("Decoded HTML Entity: !", structured_answer => any_structured_answer()),
+    'html decode &#33;' => test_zci("Decoded HTML Entity: !", structured_answer => build_structured_answer("!", "HTML Entity Decode: &amp;#33;")),
```
Comment (GuiltyDolphin) [inline t/HTMLEntitiesDecode.t]: It'd be nice if we used a `build_test` sub for calling `test_zci(build_structured_answer(@_))` (or similar).

Comment (mintsoft) [inline t/HTMLEntitiesDecode.t]: Hmm; is that the standard we're going with now? Personally I prefer it without the `build_test`

Comment (GuiltyDolphin) [inline t/HTMLEntitiesDecode.t]: @mintsoft I believe so - we have it in the templates and it is supposed to be a wrapper to let you do whatever with the test if you need to.

Comment (mintsoft) [inline t/HTMLEntitiesDecode.t]: Fair enough I'll switch it out when I get home

**Comment by GuiltyDolphin** [comment]
@mintsoft This doesn't display anything for me on `duckpan server` (triggers, but gives empty response); not sure if it's just me though - can you confirm?

**Comment by mintsoft** [comment]
@GuiltyDolphin Completely correct; the ID is wrong I think. I'm just looking at that now

**Comment by mintsoft** [comment]
@GuiltyDolphin Should be sorted now, I have no idea why I thought I needed to return options content!? 

I've removed all the `html_enc` calls so that it displays "what I want"; I think this should be safe from an XSS perspective; however I'd like signoff from staff. cc @moollaza

**Comment by moollaza** [comment]
Deploying to beta...

**Comment by moollaza** [comment]
@mintsoft Thanks! This LGTM :+1:

**Pull request merged**